cdap-data-fabric/src/main/java/co/cask/cdap/data/file/FileWriter.java

```diff
@@ -42,4 +42,11 @@
    * @throws IOException if fail to append
    */
   void appendAll(Iterator<? extends T> events) throws IOException;
+
+  /**
+   * Refreshes the FileWriter.
+   *
+   * @throws IOException if fail to refresh
+   */
+  void refresh() throws IOException;
```

Comment:
`FileWriter` will still need to implement that other interface, right?
Because `ConcurrentStreamWriter` deals with `FileWriter` interface, not the concrete class.